python/common/python/time.java

```diff
@@ -89,10 +89,30 @@ public time() {
     }
 
     @org.python.Method(
-            __doc__ = ""
+            __doc__ = "ctime(seconds) -> string\n\n"+
+                      "Convert a time in seconds since the Epoch to a string in local time.\n"+
+                      "This is equivalent to asctime(localtime(seconds)). When the time tuple is\n"+
+                      "not present, current time as returned by localtime() is used.",
+            default_args = {"seconds"}
     )
-    public static org.python.Object ctime() {
-        throw new org.python.exceptions.NotImplementedError("time.ctime() has not been implemented.");
+    public static org.python.Object ctime(org.python.Object seconds) {
+        java.util.Date date;
+        if(seconds == null){
+            long currentTimeInMillis = System.currentTimeMillis();
+            date = new java.util.Date(currentTimeInMillis);
+        } else {
+            date = new java.util.Date((long) (((org.python.types.Int) seconds.__int__()).value * 1000));
```

Comment:
Is the call to `__int__` really needed here?
Wouldn't simply `((org.python.types.Int) seconds).value` work already?